_pages/statuts.md

```diff
@@ -0,0 +1,84 @@
+---
+permalink: /statuts
+---
+
+# Statuts
+
+## Article 1er
+
+Il est fondé entre les adhérents aux présents statuts une association régie par la loi du 1^er ^juillet 1901 et le décret du 16 août 1901, ayant pour titre :
+
+**Agile France**
+
+## Article 2: "Mission"
+
+Cette association a pour but de soutenir la communauté Agile francophone et des pays de langue française, dans ses efforts pour faire évoluer l’industrie du logiciel vers plus d’efficacité et d’excellence dans les entreprises, et plus de respect des individus quel que soit leur rôle : développeur, utilisateur, client final… L’association adhère aux quatre valeurs et aux douze principes énoncées dans le Manifeste Agile.
+
+A ces fins, l’association entreprend toutes actions de promotion, de diffusion et d’éducation ayant trait aux pratiques dites Agiles et aux approches et méthodes regroupant de telles pratiques; elle intervient également pour favoriser l’évolution professionnelle des personnes utilisant ces pratiques.
+
+## Article 3: "Siège"
+
+Le siège social est fixé à "La Cantine", 151 rue Montmartre, Passage des Panoramas, 12 Galerie Montmartre, 75002 Paris.
```

Comment:
Should we update that address too?

Reply:
Je ne pense pas, c'est un document historique. Ceci dit, vu qu'il a déjà été amendé en 2012 pour changement d'adresse, je me demande si il ne l'a pas été à nouveau suite au dernier changement d'adresse.

@julienporot est-ce que le dernier envoi à la préfecture concernait aussi un amendement des statuts pour l'adresse ?

Reply:
Je confirme qu'il y avait aussi un amendement des statuts pour l'adresse.